source/server/overload_manager_impl.cc

```diff
@@ -143,6 +142,8 @@ OverloadTimerType parseTimerType(
   switch (config_timer_type) {
   case Config::HTTP_DOWNSTREAM_CONNECTION_IDLE:
     return OverloadTimerType::HttpDownstreamIdleConnectionTimeout;
+  case Config::TRANSPORT_SOCKET_CONNECT:
+    return OverloadTimerType::HttpDownstreamIdleConnectionTimeout;
```

Comment:
think you meant `TransportSocketConnectTimeout`?

Reply:
That's embarrassing, and that there wasn't a test that caught it is problematic. I've added a test to verify that the correct scale factors are applied for the different timer types.